lib/Runtime/Base/ScriptContext.h

```diff
@@ -57,6 +57,44 @@ class SRCINFO
         SRCINFO* copySrcInfo = RecyclerNew(recycler, SRCINFO, *srcInfo);
         return copySrcInfo;
     }
+
+    SRCINFO()
+    {
+        memset(this, 0, sizeof(SRCINFO));
```

Comment:
looks we don't it's allocated with RecyclerNewStructZ, let me remove this

---
In reply to: [92257043](https://github.com/Microsoft/ChakraCore/pull/2191#discussion_r92257043) [](ancestors = 92257043)